.github/workflows/build.yml

```diff
@@ -0,0 +1,124 @@
+name: Build
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+permissions:
+  contents: read
+  pull-requests: read
+
+jobs:
+  validate-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.4'
+          coverage: none
+      - name: Install dependencies
+        run: composer install
+      - name: Validate dependencies are consistent across sub-projects
+        run: ./vendor/bin/monorepo-builder validate
+
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package-path: [
+          'abstractions',
+          'authentication/phpleague',
+          'http/guzzle',
+          'serialization/json',
+          'serialization/text',
```

Comment:
Not multipart?

Reply:
Thanks for catching this! Added the lib here https://github.com/microsoft/kiota-php/pull/6

Reply:
Added in https://github.com/microsoft/kiota-php/pull/5/commits/9d97734f873911e135c85d8b185fc3eca441a424